test/tap/tests/Makefile

```
@@ -414,3 +415,20 @@ clean:
 	rm -f generate_set_session_csv set_testing-240.csv || true
 	rm -f setparser_test setparser_test2 setparser_test3  || true
 	rm -f reg_test_3504-change_user_libmariadb_helper reg_test_3504-change_user_libmysql_helper || true
+
+setparser_parsersql_test-t: setparser_parsersql_test
+	ln -fs setparser_parsersql_test setparser_parsersql_test-t
+
+setparser_parsersql_test: setparser_parsersql_test.cpp $(TAP_LDIR)/libtap.so setparser_test_common.h $(LIBPROXYSQLAR) $(LIBCOREDUMPERAR) $(PARSERSQL_LDIR)/libsqlparser.a
```

checkmake (0.3.2)

[warning] Target "setparser_parsersql_test" should be declared PHONY.

(phonydeclared)